docs/javascript/v2/features/pip-mode.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: PIP (Picture-in-Picture) Mode
+nav: 3.37
+---
+
+## Overview
+
+PiP Mode lets the user watch the room video in a small window pinned to a corner of the screen while navigating between apps or browsing content on the main screen.
+
+100ms PiP selects top 4 peers with video enabled and render them in the PiP window.
+
+<video loop="true" autoplay="autoPlay" controls="controls" id="vid" muted>
```

Comment:
the video can be much shorter. i would replace it with a screenshot of PIP - since it's a commonly known term and does not need a video walkthrough to understand. better to have a video of the code files if we want to add one

Reply:
replaced it with a screenshot.